### ISSUE TITLE
Bug Report: v3.Beta11 980 No Response

### ISSUE THREAD

**Issue by rcoletti116** [bloomkd46/homebridge-iRobot#101]
### Describe The Bug:

The Roomba does not connect. Logs show that the robot is not responding to the characteristic Gets.

### To Reproduce:

Update to latest beta, check Roomba. 

I uninstalled and reinstalled the plugin to see if that would help. No joy.

### Expected behavior:

Roomba should connect and work.

### Logs:

```shell
22/02/2022, 10:17:30] [homebridge-irobot] This plugin slows down Homebridge. The read handler for the characteristic 'On' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
[22/02/2022, 10:17:30] [homebridge-irobot] This plugin slows down Homebridge. The read handler for the characteristic 'Status Low Battery' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
[22/02/2022, 10:17:30] [homebridge-irobot] This plugin slows down Homebridge. The read handler for the characteristic 'Battery Level' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
[22/02/2022, 10:17:30] [homebridge-irobot] This plugin slows down Homebridge. The read handler for the characteristic 'Charging State' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
[22/02/2022, 10:17:30] [homebridge-irobot] This plugin slows down Homebridge. The read handler for the characteristic 'Contact Sensor State' didn't respond at all!. Please check that you properly call the callback! See https://git.io/JtMGR for more info.
```

### Config:

```shell
{
            "name": "iRobot",
            "manualDiscovery": true,
            "roombas": [
                {
                    "name": "Rosie",
                    "blid": "xxxx",
                    "password": "xxxx",
                    "ip": "xxxx"
                }
            ],
            "lowBattery": 10,
            "offAction": "pause:none",
            "status": "phase:run",
            "sensors": [
                {
                    "type": "contact"
                }
            ],
            "refreshInterval": 10,
            "platform": "iRobotPlatform"
        }
```

### Node Version:

v16.14.0

### NPM Version:

v8.5.0

### Homebridge Version:

1.4.0

### Plugin Version:

v3.0.0-beta.11

### Operating System:

Raspberian

**Comment by bloomkd46** [comment]
I see that you manually specified the IP. Are you 100% sure that it is the correct IP?

**Comment by bloomkd46** [comment]
The other thing you can try is to reboot your Roomba then reboot the plug-in

**Comment by rcoletti116** [comment]
Yes, I have a static IP set on the roomba. It is correct. 

Reboot didn't work. After a reboot of the robot and restart the logs look like this:

```shell
[22/02/2022, 10:49:05] [iRobot] [Rosie] Failed To Fetch Robot Status
 TypeError: Cannot read properties of undefined (reading 'split')
    at value (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:271:47)
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:296:91)
    at EventEmitter.emit (node:events:532:35)
    at /usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:251:25
    at runNextTicks (node:internal/process/task_queues:61:5)
    at listOnTimeout (node:internal/timers:528:9)
    at processTimers (node:internal/timers:502:7)
[22/02/2022, 10:49:06] [homebridge-irobot] This plugin threw an error from the characteristic 'Status Low Battery': Unhandled error thrown inside read handler for characteristic: undefined. See https://git.io/JtMGR for more info.
[22/02/2022, 10:49:06] [iRobot] [Rosie] Failed To Fetch Robot Status
 TypeError: Cannot read properties of undefined (reading 'split')
    at value (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:271:47)
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:296:91)
    at EventEmitter.emit (node:events:532:35)
    at /usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:229:25
    at runNextTicks (node:internal/process/task_queues:61:5)
    at listOnTimeout (node:internal/timers:528:9)
    at processTimers (node:internal/timers:502:7)
[22/02/2022, 10:49:06] [homebridge-irobot] This plugin threw an error from the characteristic 'Battery Level': Unhandled error thrown inside read handler for characteristic: undefined. See https://git.io/JtMGR for more info.
[22/02/2022, 10:49:06] [iRobot] [Rosie] Failed To Fetch Robot Status
 TypeError: Cannot read properties of undefined (reading 'split')
    at value (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:271:47)
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:296:91)
    at EventEmitter.emit (node:events:532:35)
    at /usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:240:25
    at runNextTicks (node:internal/process/task_queues:61:5)
    at listOnTimeout (node:internal/timers:528:9)
    at processTimers (node:internal/timers:502:7)
[22/02/2022, 10:49:06] [homebridge-irobot] This plugin threw an error from the characteristic 'Charging State': Unhandled error thrown inside read handler for characteristic: undefined. See https://git.io/JtMGR for more info.
[22/02/2022, 10:49:06] [iRobot] [Rosie] Failed To Fetch Robot Status
 TypeError: Cannot read properties of undefined (reading 'split')
    at value (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:271:47)
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:296:91)
    at EventEmitter.emit (node:events:532:35)
    at /usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:287:31
    at runNextTicks (node:internal/process/task_queues:61:5)
    at listOnTimeout (node:internal/timers:528:9)
    at processTimers (node:internal/timers:502:7)
[22/02/2022, 10:49:06] [homebridge-irobot] This plugin threw an error from the characteristic 'Contact Sensor State': Unhandled error thrown inside read handler for characteristic: undefined. See https://git.io/JtMGR for more info.
[22/02/2022, 10:49:06] [iRobot] [Rosie] Failed To Fetch Robot Status
 TypeError: Cannot read properties of undefined (reading 'split')
    at value (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:271:47)
    at EventEmitter.<anonymous> (/usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:296:91)
    at EventEmitter.emit (node:events:532:35)
    at /usr/local/lib/node_modules/homebridge-irobot/src/V2/platformAccessory.ts:218:25
[22/02/2022, 10:49:06] [homebridge-irobot] This plugin threw an error from the characteristic 'On': Unhandled error thrown inside read handler for characteristic: undefined. See https://git.io/JtMGR for more info.```

**Comment by rcoletti116** [comment]
I spent some time comparing the config schema for the beta 11 and latest versions. I noticed some changes. It seems when you update some of the old values can remain in the config so I cleared those out. 

I also noticed that in manual mode, when the cloud credentials are hidden in the UI they aren't actually removed from the config. I removed those manually and now I'm connected. I think this may be the key. So updating from 2.x to 3.x its best to start by clearing out the config.json and starting fresh.
